distribution/bin/enso

```diff
@@ -8,5 +8,5 @@ for opt in "$@"; do
 done
 
 
-exec java --module-path $COMP_PATH $EXTRA_OPTS $JAVA_OPTS -m org.enso.runtime/org.enso.EngineRunnerBootLoader "$@"
+exec java --module-path $COMP_PATH -Dorg.graalvm.language.enso.home=$COMP_PATH $EXTRA_OPTS $JAVA_OPTS -m org.enso.runtime/org.enso.EngineRunnerBootLoader "$@"
```

Comment:
How exactly was this working before? It this option related to `RuntimeOptions.LANGUAGE_HOME_OVERRIDE`?

Reply:
> How exactly was this working before.

In most cases, Truffle used to find the home for use. See the PR description.

> Is this option related to `RuntimeOptions.LANGUAGE_HOME_OVERRIDE`?

Slightly. `org.graalvm.language.enso.home` system property is handled by Truffle, whereas `LANGUAGE_HOME_OVERRIDE` is handled by us and overrides any previous value. `LANGUAGE_HOME_OVERRIDE` is useful for testing, where we set it to point to the micro distribution.